canonical/charmhub.io
# Refactor collaboration UI

## Done
Refactored the collaboration UI to simplify

## How to QA
- Go to https://charmhub-io-1770.demos.haus/<CHARM_NAME>/collaboration
- Check that you are able to create, revoke and resend invites

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-9050
